metals/src/main/scala/scala/meta/internal/decorations/SemanticdbTreePrinter.scala

```diff
@@ -221,4 +237,12 @@ class SemanticdbTreePrinter(
       case _ => Nil
     }
   }
+
+  private def isTuple(symbol: String) = symbol.startsWith("scala/Tuple")
+
+  private def shouldPrintTypeApply(function: s.Tree) = function match {
+    // there is no way currently to diff between Tuple2(a,b) and (a,b), so decide not to print it
```

Comment:
Thanks for the comment 😆 I can see in the future wondering... why is it like this?